node.api.js

```diff
@@ -0,0 +1,12 @@
+export default () => ({
+  webpack: config => {
+    // react-static's config.resolve.modules field is causing issues with
+    // the correct node_module resolution, so we fix that here
```

Comment:
If there's a URL(s) that discuss this as a fix / the problem maybe add in comment here?